build.sbt

```diff
@@ -67,5 +67,3 @@ lazy val testSettings = Seq(
     baseDirectory.value / "test" / "resources"
   )
 )
-
-dependencyOverrides ++= AppDependencies.overrides
```

Comment:
Why have you removed this?

Reply:
> Why have you removed this?

Do we need it? I think it's only required if we're overriding versions in AppDependencies.